secant/Dependencies/DerivationTool/DerivationToolLiveKey.swift

```diff
@@ -11,8 +11,9 @@ import ZcashLightClientKit
 extension DerivationToolClient: DependencyKey {
     static let liveValue = DerivationToolClient.live()
         
-    static func live(networkType: NetworkType = .testnet) -> Self {
-        let derivationTool = DerivationTool(networkType: networkType)
+    static func live() -> Self {
+        #if SECANT_MAINNET
+        let derivationTool = DerivationTool(networkType: .mainnet)
```

Comment:
Code for mainnet and testnet differs only in network type. Nothing else. Or Am I wrong? In this solution lot of the code is duplicated. What if you introduced variable where you store network? Like this:
```
static var networkType: NetworkType {
        #if SECANT_MAINNET
        return .mainnet
        #elseif SECANT_TESTNET
        return .testnet
        #else
        fatalError("...")
        #endif
    }
```

And then use this property in the code? This applies to all the dependencies.

Reply:
Maybe you could have this on one place and it could be used for the whole app. So there is only one place where the network type is defined.

Reply:
I created a TargetConstants enum for this and the code is much cleaner. thanks